pkg/controller/container-runtime-config/container_runtime_config_controller_test.go

```diff
@@ -111,6 +111,8 @@ func newControllerConfig(name string, platform apicfgv1.PlatformType) *mcfgv1.Co
 					PlatformStatus: &apicfgv1.PlatformStatus{
 						Type: platform,
 					},
+					APIServerURL:         fmt.Sprintf("https://api.%s.tt.testing:6443", name),
```

Comment:
How about also allowing to change the port using a var? I mean, it'll probably never change but just in case.

Reply:
this is just test data; it doesn't matter in this case.

Reply:
oh, fair enough for the tests!